source/enforce-macos-app-location.js

```diff
@@ -30,5 +30,18 @@ module.exports = () => {
 		return;
 	}
 
-	api.app.moveToApplicationsFolder();
+	api.app.moveToApplicationsFolder({
+		conflictHandler: conflict => {
+			if (conflict === 'existsAndRunning') { // Can't replace the active version of the app
+				api.dialog.showMessageBoxSync({
+					type: 'error',
+					message: `Another version of ${appName} is currently running, quit it and retry.`,
```

Comment:
"Retry" what? Not clear enough what the user is supposed to do.

```suggestion
					message: `Another version of ${appName} is currently running. Quit the running app and then launch this version of the app again.`,
```

How about this?

Reply:
"quit it" seems fairly clear to me, expanding it to "Quit the running app" sounds too verbose.

"retry" can be definitely more obscure than "then launch this version of the app again", but it's also a lot less text to read 🤔

Maybe a middle ground "Another version of ${app.getName ()} is currently running. Quit it, then launch this version of the app again." would work best.

Let's go with whatever you like best.

Reply:
> Maybe a middle ground "Another version of ${app.getName ()} is currently running. Quit it, then launch this version of the app again." would work best.

👍